### ISSUE TITLE
scales="free" on facet_grid no longer has any effect on `y` axis within rows

### ISSUE THREAD

**Issue by cboettig** [hadley/ggplot2#971]
(**Updated to clarify version**): Using CRAN version 1.0.0 or CRAN archive 0.9.3.1, I'm finding that the `scales="free"` argument has no effect on `y` axis scaling.  I try the example from the documentation: 

```s
mt <- ggplot(mtcars, aes(mpg, wt, colour = factor(cyl))) + geom_point()
mt + facet_grid(. ~ cyl, scales = "free")
```

and I get: 

![notfree](https://cloud.githubusercontent.com/assets/222586/3180935/22055af6-ec38-11e3-9a27-31ce6243c5a4.png)

```s
sessionInfo()
```

```
R version 3.1.0 (2014-04-10)
Platform: x86_64-pc-linux-gnu (64-bit)

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=C                  LC_COLLATE=C              
 [5] LC_MONETARY=C              LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices datasets  utils     methods   base     

other attached packages:
[1] ggplot2_1.0.0 knitr_1.6     devtools_1.5 

loaded via a namespace (and not attached):
 [1] MASS_7.3-33       RCurl_1.95-4.1    Rcpp_0.11.1       colorspace_1.2-4 
 [5] digest_0.6.4      evaluate_0.5.5    formatR_0.10      grid_3.1.0       
 [9] gtable_0.1.2      httr_0.3          memoise_0.2.1     munsell_0.4.2    
[13] parallel_3.1.0    plyr_1.8.1        proto_0.3-10      reshape2_1.4.0.99
[17] scales_0.2.4      stringr_0.6.2     tools_3.1.0       whisker_0.3-2    
```

**Comment by hadley** [comment]
Did free y scales ever work like this? I think you want `facet_wrap()`.

**Comment by cboettig** [comment]
I think so too :-). thanks, I shoulda closed this a while ago.

**Comment by xfim** [comment]
Solved in commit 1fba535b06fc2 by @lionel-
